source/sdk/math.ooc

```diff
@@ -191,18 +191,23 @@ extend Float {
 	}
 	toRadians: func -> This { This pi / 180.0f * this }
 	toDegrees: func -> This { 180.0f / This pi * this }
-	maximum: static func (first, second: This) -> This {
-		first > second ? first : second
-	}
-	minimum: static func (first, second: This) -> This {
-		first < second ? first : second
-	}
 	modulo: func (divisor: This) -> This {
 		result := this
 		if (result < 0)
 			result += ((result abs()) / divisor) ceil() * divisor
 		result mod(divisor)
 	}
+	equals: func (other: This, tolerance := This defaultTolerance) -> Bool { (this - other) abs() < tolerance }
+	lessThan: func (other: This, tolerance := This defaultTolerance) -> Bool { this < other && !this equals(other) }
+	greaterThan: func (other: This, tolerance := This defaultTolerance) -> Bool { this > other && !this equals(other) }
+	lessOrEqual: func (other: This, tolerance := This defaultTolerance) -> Bool { !this greaterThan(other) }
+	greaterOrEqual: func (other: This, tolerance := This defaultTolerance) -> Bool { !this lessThan(other) }
```

Comment:
Maybe we can move these to the `LLong` cover in `Numbers.ooc`, and then we don't need to include them in Float, Double and LDouble.